main.py

```diff
@@ -479,25 +490,27 @@ def main(args):
                       'duration': duration_to_string(selector.duration),
                       'site': selector.site,
                       'client_provider': selector.client_provider,
-                      'metric': selector.metric,
-                      'mlab_project': selector.mlab_project,
+                      'client_country': selector.client_country,
+                      'metric': selector.metric
                     }
-    thread_metadata['data_filepath'] = build_filename('data',
-                                                      args.output,
-                                                      thread_metadata['date'],
-                                                      thread_metadata['duration'],
-                                                      thread_metadata['site'],
-                                                      thread_metadata['client_provider'],
-                                                      thread_metadata['metric'])
+    data_filepath = build_filename('data',
+                                      args.output,
```

Comment:
Should be aligned with the opening paren:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Indentation#Indentation